src/modules/solvers/QiskitQAOA.py

```diff
@@ -179,16 +184,23 @@ def run(self, mapped_problem: any, device_wrapper: any, config: Config, **kwargs
         t = mapped_problem['t']
         start = start_time_measurement()
         ising_op = self._get_pauli_op((t, J))
+        eagle_max_iter = 1
```

Comment:
I think this should not be hard-coded. Instead, it should be part of the device configuration.